autogpt_platform/backend/backend/executor/manager.py

```diff
@@ -952,31 +1018,112 @@ def run_service(self):
         logger.info(f"[{self.service_name}] ⏳ Connecting to Redis...")
         redis.connect()
 
-        sync_manager = multiprocessing.Manager()
+        logger.info(f"[{self.service_name}] Ready to consume messages...")
         while True:
-            graph_exec_data = self.queue.get()
-            graph_exec_id = graph_exec_data.graph_exec_id
-            logger.debug(
-                f"[ExecutionManager] Dispatching graph execution {graph_exec_id}"
+            # cancel graph execution requests
+            method_frame, _, body = channel.basic_get(
+                queue=GRAPH_EXECUTION_CANCEL_QUEUE_NAME,
+                auto_ack=True,
```

Comment:
nit: document that auto_ack is a shittly named arg

Reply:
What's so shit about it? 🥺 
The name sounds self-explanatory to me.

Reply:
It’s behavior is weird compared to the name acto_ack being true means you have to manually ack or nack which is crazy